packages/format-currency/src/index.ts

```diff
@@ -372,8 +417,20 @@ function getSmallestUnitDivisor( precision: number ): number {
 	return 10 ** precision;
 }
 
+interface GeoLocationResponse {
+	country_short: string;
+}
+
+function isGeolocationResponse( response: unknown ): response is GeoLocationResponse {
```

Comment:
Should this be named `containsGeolocationCountry` or something similar? Being explicit wins IMHO. Same for the `GeoLocationResponse` type guard, since it only contains a `country_short` anyway, and the geolocation response has a bunch more fields we don't need to think about here.

Reply:
👍